config.js

```diff
@@ -68,7 +68,7 @@ const config = {
   SIMPLE_SITE: process.env.SIMPLE_SITE === 'true',
   SHOW_ADS: process.env.SHOW_ADS === 'true',
   KNOWN_APP_DOMAINS: process.env.KNOWN_APP_DOMAINS ? process.env.KNOWN_APP_DOMAINS.split(',') : [],
-  STRIPE_PUBLIC_KEY: process.env.STRIPE_PUBLIC_KEY,
+  STRIPE_PUBLIC_KEY: 'pk_test_NoL1JWL7i1ipfhVId5KfDZgo' || process.env.STRIPE_PUBLIC_KEY,
```

Comment:
Merge cleanup reminder.
To avoid having to do this cleanup, we can override this in a local `.env` file (and also in dev instances), so we don't need to worry whether this gets accidentally merged in or not.